packages/module/patternfly-docs/content/examples/basic.md

```diff
@@ -56,4 +56,4 @@ The following example demonstrates how to use the `async` function to send data
 To customize the content displayed in the feedback modal, pass in a custom `i18n` object with values for each property of`<FeedbackModal>`. This allows you to prepare a modal for different languages and requirements.
```

Comment:
```suggestion
To customize the content displayed in the feedback modal, pass in a custom `i18n` object with values for each property of `<FeedbackModal>`. This allows you to prepare a modal for different languages and requirements.
```

Just noticed there is a space missing here, so maybe this can be the whitespace change needed to trigger a release? Also I'm not sure if this repo uses semantic release messages, but if it does `chore()` will not initiate a release. `fix()` will issue a patch release (eg 1.2.3 -> 1.2.4) and `feat()` will issue a minor/feature release (eg 1.2.3 -> 1.3.0)

Reply:
It does use semantic release. Can you update the git commit to use `fix:` Other then @mcoker comment. lgtm. Thanks @parthivrh

Reply:
@mcoker @dlabaj this is one of the repos where I have [some custom release rules in the semantic release config file](https://github.com/patternfly/react-user-feedback/blob/main/packages/module/release.config.js) so `chore(ci-release)` _should_ issue a patch release.

This _is_ semantic-release though so anything is possible.